### ISSUE TITLE
Question about 2D denseunet training

### ISSUE THREAD

**Issue by easycui** [xmengli999/H-DenseUNet#38]
In the code, it shows that 2D denseunet will train 6000 epochs without validation set. My question is when to stop the training. I guess that training the model for 6000 epochs and pick the one that has lowest training loss, it that correct? Thanks.

**Comment by xmengli999** [comment]
I use the last model.

**Comment by easycui** [comment]
Thanks
